File: data_structures_by_c++/linked_list/circular_linked_list/create.cpp
#include <bits/stdc++.h>
using namespace std;

class Node
{
public:
    int val;
    Node *next;
    Node(int val)
    {
        this->val = val;
        this->next = NULL;
    }
};

int main()
{
    Node *head = new Node(10);
    Node *a = new Node(20);
    Node *b = new Node(30);

    head->next = a;
    a->next = b;
    b->next = head;

    Node *tmp = head;
    for (int i = 0; i < 3; i++)
    {
        cout << tmp->val << " ";
        tmp = tmp->next;
    }
    cout << endl;
}

File: data_structures_by_python/linked_list/circular_linked_list/create.py
class Node:
    def __init__(self, val):
        self.val = val
        self.next = None


head = Node(10)
a = Node(20)
b = Node(30)

head.next = a
a.next = b
b.next = head

tmp = head
for i in range(3):
    print(tmp.val, end=" ")
    tmp = tmp.next
print()
